x/valset-pref/client/cli/tx.go

```diff
@@ -19,7 +19,8 @@ func GetTxCmd() *cobra.Command {
 	txCmd := osmocli.TxIndexCmd(types.ModuleName)
 	osmocli.AddTxCmd(txCmd, NewSetValSetCmd)
 	osmocli.AddTxCmd(txCmd, NewDelValSetCmd)
-	osmocli.AddTxCmd(txCmd, NewUnDelValSetCmd)
+	//osmocli.AddTxCmd(txCmd, NewUnDelValSetCmd)
```

Comment:
Can we include TODOs to make it clear that these are intentionally commented?

Reply:
TODOs included https://github.com/osmosis-labs/osmosis/pull/6685/commits/fdaa8166419ee90e6cbbe1d207446c15ec01b5a0